### DISCUSSION TITLE
COMBLE-MIP: permission denied when trying to create Jupyter Notebook (Jupyter Hub at ARM)

### DISCUSSION THREAD

**Discussion by AlejandroBaroPerez** [ARM-Development/comble-mip#46] [General]
Hello, I have applied for elevated Jupyter Hub access at ARM. Now I can open the Jupyter Hub platform at ARM but I get "Permission denied" when I try to create a Jupyter Notebook. Can you let me know how to solve this issue? Thanks in advance. Here is the screenshot: 

![Screenshot](https://github.com/ARM-Development/comble-mip/assets/52293043/4b8237ca-c83f-4f82-8143-ff6a48921bb7)

**Comment by twjuliano** [comment]
Hello @AlejandroBaroPerez,
Thanks for letting us know about your issue. It looks like you will need to apply for elevated access. Please follow the instructions on [this page](https://arm-development.github.io/comble-mip/notebooks/howto/workbench_instructions.html) to do so. I think you should receive access within a couple of days.
Thank you,
Tim

**Reply by AlejandroBaroPerez** [reply]
Ok thanks @twjuliano, I applied for the elevated access. Before, I had only created my ARM account, but for some reason I thought the elevated Jupyter access was included in the account creation. Thanks!

**Comment by twjuliano** [comment]
Thank you for confirming, @AlejandroBaroPerez!
